<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTestSettings</class>
 <widget class="QDialog" name="MainTestSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Параметры тестирования</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Непрерывный тест</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Время прохода (в одну сторону):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTimeEdit" name="timeEdit">
           <property name="maximumTime">
            <time>
             <hour>0</hour>
             <minute>15</minute>
             <second>0</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>0</hour>
             <minute>3</minute>
             <second>0</second>
            </time>
           </property>
           <property name="displayFormat">
            <string>mm:ss</string>
           </property>
           <property name="time">
            <time>
             <hour>0</hour>
             <minute>10</minute>
             <second>0</second>
            </time>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_cyclic_test">
           <property name="text">
            <string>Циклический тест</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Количество циклов:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_num_cycles">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Время выполнения цикла:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QTimeEdit" name="timeEdit_cycle_time">
           <property name="date">
            <date>
             <year>2000</year>
             <month>1</month>
             <day>2</day>
            </date>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>5</minute>
             <second>0</second>
             <year>2000</year>
             <month>1</month>
             <day>2</day>
            </datetime>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>5</second>
             <year>2000</year>
             <month>1</month>
             <day>2</day>
            </datetime>
           </property>
           <property name="maximumDate">
            <date>
             <year>2000</year>
             <month>1</month>
             <day>2</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>2000</year>
             <month>1</month>
             <day>2</day>
            </date>
           </property>
           <property name="maximumTime">
            <time>
             <hour>0</hour>
             <minute>5</minute>
             <second>0</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>5</second>
            </time>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::Section::MinuteSection</enum>
           </property>
           <property name="displayFormat">
            <string>mm:ss</string>
           </property>
           <property name="time">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>10</second>
            </time>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Тест с задержкой</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Время отклика:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_response">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
         </property>
         <property name="minimum">
          <double>0.050000000000000</double>
         </property>
         <property name="maximum">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Время задержки:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_delay">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>6.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Количество точек:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_points">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>150.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Минимальный сигнал:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_signal_min">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>3.000000000000000</double>
         </property>
         <property name="maximum">
          <double>21.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Максимальный сигнал:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_signal_max">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>3.000000000000000</double>
         </property>
         <property name="maximum">
          <double>21.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>21.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="text">
        <string>Начать тест</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>checkBox_cyclic_test</tabstop>
  <tabstop>spinBox_num_cycles</tabstop>
  <tabstop>timeEdit_cycle_time</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_start</tabstop>
  <tabstop>doubleSpinBox_response</tabstop>
  <tabstop>doubleSpinBox_delay</tabstop>
  <tabstop>doubleSpinBox_points</tabstop>
  <tabstop>doubleSpinBox_signal_min</tabstop>
  <tabstop>doubleSpinBox_signal_max</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_start</sender>
   <signal>clicked()</signal>
   <receiver>MainTestSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainTestSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
